### PR TITLE
[DEV-3779] Fix for Active and Public Campuses

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?

The purpose of this PR is to get only the campuses that are active and public. 

### How do I test this PR?

Only active and public campuses should show in Onboarding.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [DEV-3779](https://newspring.atlassian.net/browse/DEV-3779)
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android if applicable
- [ ] Set two relevant reviewers

## REVIEW

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by IsaacHardy — CHANGES_REQUESTED** [review]
We need to write a test for the campuses resolver.

**Comment by redreceipt** [comment]
@IsaacHardy not sure how to make the snap better, could use your help

File: packages/apollos-church-api/src/data/campuses/__tests__/__snapshots__/resolvers.tests.js.snap [hunk 1]
```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Campus gets campuses 1`] = `
+Object {
+  "data": Object {
+    "campuses": null,
+  },
+  "errors": Array [
+    [GraphQLError: Cannot read property 'value' of undefined],
```
Comment (IsaacHardy) [inline packages/apollos-church-api/src/data/campuses/__tests__/__snapshots__/resolvers.tests.js.snap]: I don't think this is what you want returned in your resolver.

**Review by IsaacHardy — CHANGES_REQUESTED** [review]

File: packages/apollos-church-api/src/data/campuses/__tests__/__snapshots__/resolvers.tests.js.snap (hunk 1 above)
Comment (redreceipt) [reply packages/apollos-church-api/src/data/campuses/__tests__/__snapshots__/resolvers.tests.js.snap]: hence my comment above

File: packages/apollos-church-api/src/data/campuses/resolver.js [hunk 2]
```diff
@@ -3,15 +3,13 @@ import { schemaMerge } from '@apollosproject/server-core';
 import { createAssetUrl } from '../utils';
 
 const resolver = {
+  Query: {
+    campuses: (root, { location }, { dataSources }) =>
+      dataSources.Campus.getPublicByLocation(location),
+  },
   Campus: {
-    latitude: ({ location }) =>
```
Comment (IsaacHardy) [inline packages/apollos-church-api/src/data/campuses/resolver.js:7]: Why were these properties removed?

Comment (redreceipt) [reply packages/apollos-church-api/src/data/campuses/resolver.js:7]: I don't believe we need them. It just inherits from core. And we don't need all the extra default checking per our conversation from earlier today.

**Review by IsaacHardy — APPROVED** [review]
LGTM

**Pull request merged**